Makefile

```diff
@@ -60,6 +60,8 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	dpkg-source -x $(DSC_FILE)
 
 	pushd $(BUILD_DIR)
+	sed -i '/abiname:.*/ s/$$/-1/' debian/config/defines
```

Comment:
I remember for debian kernel build, if there is any change with kernel/user ABI,  some ABI function file has to be updated explicitly.   Did you change any of the ABI files?

Reply:
Not that I am aware of. I did not make any change explicitly.

Reply:
@jipanyang , check my patch and change the abi to 8-2